### PR TITLE
Allow app-viewers to invite others as app-viewers

### PR DESCRIPTION
**Pull request by sharat87**

### CODE REVIEW

**Comment by sharat87** [comment]
Aside, genuinely impressed at how little effort a change like this took. It's been a while since I looked at the ACL/roles related code, so wasn't following well enough. Great done @trishaanand :)

**Comment by nidhi-nair** [comment]
> Aside, genuinely impressed at how little effort a change like this took. It's been a while since I looked at the ACL/roles related code, so wasn't following well enough. Great done @trishaanand :)

Agreed, this looks beautiful!

File: app/server/appsmith-server/src/main/java/com/appsmith/server/acl/AppsmithRole.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
             Set.of(MANAGE_ORGANIZATIONS, ORGANIZATION_INVITE_USERS)),
     ORGANIZATION_DEVELOPER("Developer", "Can edit and view applications along with inviting other users to the organization",  Set.of(READ_ORGANIZATIONS,
             ORGANIZATION_MANAGE_APPLICATIONS, ORGANIZATION_READ_APPLICATIONS, ORGANIZATION_PUBLISH_APPLICATIONS, ORGANIZATION_INVITE_USERS)),
-    ORGANIZATION_VIEWER("App Viewer", "Can only view applications",  Set.of(READ_ORGANIZATIONS, ORGANIZATION_READ_APPLICATIONS));
+    ORGANIZATION_VIEWER("App Viewer", "Can only view applications",  Set.of(READ_ORGANIZATIONS, ORGANIZATION_READ_APPLICATIONS, ORGANIZATION_INVITE_USERS));
```
Comment (nidhi-nair) [inline app/server/appsmith-server/src/main/java/com/appsmith/server/acl/AppsmithRole.java:26]: Very minor comment, could you please update the description to reflect the change in roles?

Comment (sharat87) [reply app/server/appsmith-server/src/main/java/com/appsmith/server/acl/AppsmithRole.java:26]: Oh yeah, I didn't realize the other descriptions mentioned the capability to invite. Doing this now.

**Review by nidhi-nair — APPROVED** [review]

File: app/server/appsmith-server/src/main/java/com/appsmith/server/acl/AppsmithRole.java (hunk 1 above)
Comment (sharat87) [reply app/server/appsmith-server/src/main/java/com/appsmith/server/acl/AppsmithRole.java]: Pushed. Sorry, just saw your change suggestion. :|

**Comment by sharat87** [comment]
Converting to draft since I just realized this needs a migration to add the invite policy to all current app-viewers.

**Comment by sharat87** [comment]
@nidhi-nair @trishaanand Added migration. Can (one of) you folks take a look at the migration function and approve please? Thanks!

**Review by trishaanand — APPROVED** [review]

**Pull request merged**